pandas/core/series.py

```diff
@@ -1429,8 +1429,52 @@ def mode(self):
         # TODO: Add option for bins like value_counts()
         return algorithms.mode(self)
 
-    @Appender(base._shared_docs['unique'] % _shared_doc_kwargs)
     def unique(self):
+        """
+        Return unique values of Series object.
+
+        Uniques are returned in order of appearance. Hash table-based unique,
+        therefore does NOT sort.
+
+        Returns
+        -------
+        unique values : Series or Categorical
```

Comment:
It's never a Series, only an numpy array or Categorical. I would also include something about that like

```
ndarray or Categorical
    The unique values returned as a NumPy array. In case of categorical data type, returned as a Categorical
```